cate/ds/local.py

```diff
@@ -151,8 +151,9 @@ def open_dataset(self,
                      protocol: str = None,
                      monitor: Monitor = Monitor.NONE) -> Any:
         time_range = TimeRangeLike.convert(time_range) if time_range else None
-        if var_names:
-            var_names = VarNamesLike.convert(var_names)
+        region = PolygonLike.convert(region) if region else None
+        var_names = VarNamesLike.convert(var_names) if var_names else None
```

Comment:
`if var_names else None` is redundant, please see code of `VarNamesLike.convert`